horace_core/spaghetti_plot.m

```diff
@@ -250,11 +261,9 @@
 
     end
 end
-
-
 if ~isempty(plane_normal)
-    fprintf('spaghetti_plot: rlp found to lie in the plane perpendicular to (%g %g %g)\n', ...
-            sqw_proj.transform_img_to_hkl(plane_normal'));
+    fprintf(2,'*** spaghetti_plot: rlp found to lie in the plane perpendicular to (%g %g %g)\n', ...
```

Comment:
Is there a reason this should be printed to (presumably) stderr?

Reply:
yes -- generally this is error or warning as spaghetti plot's point do not lie in one plane. If they are, something is usually wrong with your data from scientific point of view Programmatically ok, but if you've bothered to organize  the check for it not to lie in one plane, its probably worth highlighint this in the warning.